model/import/ChecksumGenerator.php

```diff
@@ -37,9 +37,13 @@ public function __construct(ListService $listService)
     public function getRangeChecksum(Property $property): string
     {
         $labels = [];
-        foreach ($this->listService->getListElements($property->getRange()) as $listEntry) {
-            $labels[] = strtolower($listEntry->getLabel());
+
+        if ($property->getRange()) {//@TODO @FIXME Validate the fix with Bart before merge
```

Comment:
- [ ] Revert this change after proper fix is addressed by authoring team

Reply:
Here is PR that will prevent passing property without range
https://github.com/oat-sa/extension-tao-itemqti/pull/2587